examples/MinioClientWithClientGrantsProvider.java

```diff
@@ -72,42 +70,20 @@ public static void main(String[] args) throws Exception {
         "https://IDP-HOST:IDP-PORT/auth/realms/master/protocol/openid-connect/token";
 
     // Client-ID to fetch JWT.
-    String clientId = "USER-ID";
+    String clientId = "CLIENT-ID";
 
     // Client secret to fetch JWT.
-    String clientSecret = "PASSWORD";
-
-    // Client-ID of MinIO service on IDP.
-    String idpClientId = "MINIO-CLIENT-ID";
+    String clientSecret = "CLIENT-SECRET";
 
     // STS endpoint usually point to MinIO server.
     String stsEndpoint = "http://STS-HOST:STS-PORT/";
 
-    // Policy to Credentials.
-    String policy =
```

Comment:
Why do we remove this? Doesn't minio server support it?

Reply:
Inline policy is supported and its optional in the API. I can bring it back.